base/base/find_symbols.h

```diff
@@ -205,6 +205,16 @@ inline const char * find_first_symbols_sse2(const char * const begin, const char
     return return_mode == ReturnMode::End ? end : nullptr;
 }
 
+template <bool positive, ReturnMode return_mode, char... symbols>
+inline const char * find_first_symbols_sse2_markdown(const char * const begin, const char * const end)
```

Comment:
It's strange that we "pollute" such a nice, pure, and generic `find_symbols` header with such a dirty, ad-hoc, and miserable thing as Markdown.